app/src/androidTest/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreenAlertDialogTest.kt

```diff
@@ -98,7 +90,7 @@ class ParkingDetailsScreenAlertDialogTest {
       ParkingDetailsAlertDialogShowImage(
           parkingViewModel,
           userViewModel,
-          onDismiss = { dismissedCalled = true },
+          onDismiss = {},
```

Comment:
why was this removed?

Reply:
I removed `dismissedCalled` a few line above because it was not used in the test so it was useless. This var was never asserted.